bundles/io/org.eclipse.smarthome.io.rest.test/src/test/groovy/org/eclipse/smarthome/io/rest/test/filter/CorsFilterTest.groovy

```diff
@@ -34,6 +34,8 @@ class CorsFilterTest {
     private static final String HTTP_GET_METHOD = "GET"
     private static final String HTTP_OPTIONS_METHOD = "OPTIONS"
 
+    private static final String CONTENT_TYPE_HEADER = "content-type";
```

Comment:
I don't think we should have this method. Header names are case-insensitive as described by the specifications (https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2).
Although We can change `content-type` casing because `Content-Type` is more commonly written as you stated and also here (https://www.w3.org/Protocols/rfc1341/4_Content-Type.html).